clients/drcachesim/drpt2trace/pt2ir.cpp

```diff
@@ -399,9 +418,16 @@ pt2ir_t::convert(DR_PARAM_IN const uint8_t *pt_data, DR_PARAM_IN size_t pt_data_
                 }
 #endif
             }
+            ++decoded_instr_count;
             drir->append(instr, instr_ip, insn.size, insn.raw);
         }
     }
+    VPRINT(1,
+           "libipt decoded " UINT64_FORMAT_STRING
+           " instructions with " UINT64_FORMAT_STRING " recoverable errors.\n",
```

Comment:
Same comments about "recoverable errors" not being clear and not giving the user any idea of what might or might not be wrong with the final trace.

Reply:
I changed it to "non-fatal errors". Note that not all error instances correspond to one PC discontinuity.